src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
```

Comment:
- [ ] we can remove the code from App.js component and make it to return a hello world message. Some like this:
```JS
function App() {
  return (
    <div className="App">
        <h1>Hello World</h1>
    </div>
```

- [ ] After doing that we can delete the `logo.svg` file, since it's not going to be used by us. We need also to delete its import at the App.js file.